releases/katello/4.10/settings

```diff
@@ -0,0 +1,4 @@
+FULLVERSION='4.10.0'
```

Comment:
This implies it's GA, but the full version includes the RC part. See dbc04052116c8e58d1323b9894444a0802f5db3f

Reply:
@ekohl fixed, sorry about that I should have looked at a rc1 commit before just blindly copy/pasta

Reply:
For Foreman's branching procedure we have explicit instructions: https://github.com/theforeman/theforeman-rel-eng/blob/5f6403570c60b7144168439562d9b5a3d722de09/procedures/foreman/branch.md.erb#L62

Katello should have a similar step, which would prevent any future confusion. Could you submit a PR?

Reply:
Yeah will make a follow up today